bears/swift/TailorBear.py

```diff
@@ -0,0 +1,80 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.results.Result import Result
+from coalib.settings.Setting import path
+
+
+@linter(executable="tailor",
+        output_format=None,
+        prerequisite_check_command=("tailor", "-v"),
+        prerequisite_check_fail_message="Tailor is not installed. Refer "
+        "https://tailor.sh/ for installation details.")
+class TailorBear:
+    """
+    Analyze Swift code using ``tailor`` and check for code style related
```

Comment:
remove `using tailor`, the user shouldn't even know this on an ideal wrapper, the last line is only because we don't have enough documentation and so we link to theirs